source/includes/_invoice-payment.md

```diff
@@ -349,17 +364,29 @@ KillBillClient::Model::InvoicePayment.chargeback(payment_id,
 ```python
 invoicePaymentApi = killbill.InvoicePaymentApi()
 
-payment_id = '2276b3c9-4e51-41b2-b5bf-9ddc11582ee4'
+paymentId = '2276b3c9-4e51-41b2-b5bf-9ddc11582ee4'
 
-transactionBody = killbil.PaymentTransaction(amount=50.0, currency='USD')
+transactionBody = killbill.InvoicePaymentTransaction(amount=50.0, currency='USD')
```

Comment:
Same comment as above, drop `killbill.`  here and in other places as applicable to keep the code consistent with other Python snippets.